libs/vkd3d/command.c

```diff
@@ -6353,6 +6363,7 @@ static void STDMETHODCALLTYPE d3d12_command_list_CopyResource(d3d12_command_list
     d3d12_command_list_track_resource_usage(list, src_resource, true);
 
     d3d12_command_list_end_current_render_pass(list, false);
+    d3d12_command_list_end_batch(list);
```

Comment:
It would be clearer to just call this transfer_batch rather than just batch. Easier to read the code that way I think.